www/src/assets/mobile-nav-icons.js

```diff
@@ -3,5 +3,13 @@ import DocsIcon from "!raw-loader!./docs.svg"
 import TutorialIcon from "!raw-loader!./tutorial.svg"
 import PluginsIcon from "!raw-loader!./plugins.svg"
 import ShowcaseIcon from "!raw-loader!./showcase.svg"
+import EcosystemIcon from "!raw-loader!./ecosystem.svg"
```

Comment:
Do we still need the `raw-loader` for SVGs? I've been importing them without it in my V2 sites and everything works as expected. (This may be out-of-scope for this PR, but if we can eliminate extra config like this, I'd like to put it on the roadmap.)

Reply:
In fact, this appears to have worked since V1 without the `raw-loader`: https://github.com/jlengstorf/lengstorf.com/blob/9ef7786073c1f586b91b967c67f9706102c77ee1/src/pages/about.js#L6

Reply:
Yeah, I did not think only conformed to the existing code.

Reply:
@jlengstorf I've checked it out. It works on your site because you are using svg as  `src` of  `img`, we on www use it as an inline `svg`, and we still need the loader, without it we get `data:image...`